sphinx_gallery/utils.py

```diff
@@ -151,3 +154,21 @@ class Bunch(dict):
     def __init__(self, **kwargs):  # noqa: D102
         dict.__init__(self, kwargs)
         self.__dict__ = self
+
+
+def _has_pypandoc(raise_error=False):
```

Comment:
`raise_error` seems like an inappropriate name, it implies that a `raise ...Error` will happen if something goes wrong. maybe `log_result=True`?

But really if you do the refactoring I suggest, you will only ever use this once so you can eliminate the `kwarg` altogether.

Reply:
Changed to:

> Seeing this more than one place makes it seem like we should have a _get_pandoc() for example that either returns pypandoc, version or None, None (DRY)